src/app/components/JumpTo/index.stories.tsx

```diff
@@ -17,11 +25,26 @@ export default {
   },
 };
 
+export const ExampleWithRelatedContentLink = () => {
+  const jumpToBlock =
+    pidginArticleFixtureWithJumpToBlock.data.article.content.model.blocks.find(
+      block => block.type === 'jumpTo',
+    );
```

Comment:
Is it worth moving this to outside the stories (as it is duplicated here and in the Example story) - then you can reuse it in both?

Reply:
Good idea. Done!